### ISSUE TITLE
Somehow widgets seem to interfere with each other

### ISSUE THREAD

**Issue by jasperhartong** [jasperhartong/scriptable-ts-boilerplate#1]
If more than 1 `WidgetLoader`-based widget is loaded, the following error sometimes appears on either one of them:

`Call Script.setWidget() to set content of the widget`

![IMG_0C04168BAC5E-1](https://user-images.githubusercontent.com/47074382/97887373-1ac6cc00-1d2a-11eb-932c-0f1805e775b6.jpeg)

I though that rolling-up the components with `iife` would ensure that no global state is shared. But I get the feeling it does..

Not sure how to debug this also :)

**Comment by jasperhartong** [comment]
Seems to happen after I've called `widget.presentSmall()` while it's also a widget on the homescreen.

Steps to reproduce:

0. visit https://scriptable-ts-boilerplate.vercel.app/ / (or local)
1. Add Stickynote widget to Scriptable and to home screen
2. Add COVID note widget to Scriptable and to home screen
3. Run Stickynote from within Scriptable
Result: Scriptable shows preview in app

5. Go to home screen
Result: The widget now shows the above error.

Must have something to do then with the following code:

https://github.com/jasperhartong/scriptable-ts-boilerplate/blob/eaf977cdbe7923a7df3fb4a9702feca09c4149ae/scriptable-api/public/compiled-widgets/widgetLoader.js#L71-L76

Tried:
- removing the `await`
- making it an `If ... else` (so either calling presentSmall OR setWidget)
